### ISSUE TITLE
Not sure of APCA to BPCA conversion is accurate

### ISSUE THREAD

**Issue by pawn002** [pawn002/color-pair-quick-iterator#14]
refs #12 

Had to create own method of 'scaling' APCA scores to WCAG-ish scores because npm package `bridge-pca` is unusable off-the-shelf.

Time to vette the conversions

**Comment by pawn002** [comment]
### Issue Spotted

#928f8f on #ffffff generates an APCA score of 59.308 but the BPCA score is 3

_The BPCA score should be less than 3._
